include/velocypack/Slice.h

```diff
@@ -621,45 +677,45 @@ class Slice {
     if (!isUTCDate()) {
       throw Exception(Exception::InvalidValueType, "Expecting type UTCDate");
     }
-    uint64_t v = readIntegerFixed<uint64_t, sizeof(uint64_t)>(_start + 1);
+    uint64_t v = readIntegerFixed<uint64_t, sizeof(uint64_t)>(valueStart() + 1);
     return toInt64(v);
   }
 
   // return the value for a String object
   char const* getString(ValueLength& length) const {
-    uint8_t const h = head();
+    uint8_t const h = valueHead();
```

Comment:
I think this should still operate on `head()` and not on `valueHead()`.
If a client application needs to peek into the value behind the tag, I think it should explicitly resolve the tag.